app/src/main/java/com/khaithu/a3ctech_android_final_project/sharedprefences/DataLocalManager.kt

```diff
@@ -1,15 +1,18 @@
 package com.khaithu.a3ctech_android_final_project.sharedprefences
 
 import android.content.Context
+import com.khaithu.a3ctech_android_final_project.model.enum.LanguageEnum
 
 class DataLocalManager {
     private lateinit var mySharedPreference: MySharedPreference
 
     companion object {
         private lateinit var instance: DataLocalManager
+        private const val LANGUAGE = "LANGUAGE"
+        private const val STATUS_LOGIN = "STATUS_LOGIN"
```

Comment:
```suggestion
        private const val LANGUAGE_KEY = "LANGUAGE_KEY"
        private const val LOGIN_STATUS_KEY = "LOGIN_STATUS_KEY"
```
or define at new class SharedPrefsKey